test/JDBC/expected/babel_4328_datetime-before-14_12-vu-verify.out

```diff
@@ -605,7 +605,7 @@ select cast('4:12:12:1234' as datetime)
 go
 ~~ERROR (Code: 33557097)~~
 
-~~ERROR (Message: data precision out of range for datetime)~~
+~~ERROR (Message: invalid input syntax for type datetime: "4:12:12:1234")~~
```

Comment:
Have we verified if this is the error message thrown by TSQL?

Reply:
yes